### ISSUE TITLE
[3.0.0-beta.7] Cannot add instrumentations anymore

### ISSUE THREAD

**Issue by mderriey** [microsoft/ApplicationInsights-node.js#1182]
Hi there,

With 3.0.0-beta.7 consuming @azure/monitor-opentelemetry, we can't add additional instrumentations, which is a step back feature-wise.

Nothing is exposed on [the `TelemetryClient` class](https://github.com/microsoft/ApplicationInsights-node.js/blob/beta/src/shim/telemetryClient.ts), and [the `AzureMonitorOpenTelemetryClient` class](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-opentelemetry/src/client.ts) doesn't expose its `TraceHandler` instance.

I suspect this should be an issue on the azure-sdk-for-js repo, but I thought I'd open it here in case you have plans to expose it somehow through `TelemetryClient`.

Cheers.

**Comment by cuzzlor** [comment]
I came here wondering if the `[Object object]` output has been fixed, I guess beta.7 is not the one 😭

**Comment by gidich** [comment]
@mderriey - Thank you for posting this issue - I'm hoping that functionality will return in a future beta, for now, we'll be staying at beta.6.

**Comment by heruwala** [comment]
I had to downgrade to beta.6. Please allow adding instrumentations.

**Comment by hectorhdzg** [comment]
Instrumentation can still be added in beta.7, the way to do it is slightly different, we are going though some refactoring and API reviews that will change how this SDK is consumed, but features should continue to exist.

```ts
const { ApplicationInsightsClient } = require("applicationinsights");
const { trace } = require('@opentelemetry/api');
const { registerInstrumentations } = require("@opentelemetry/instrumentation");
const { WinstonInstrumentation } = require("@opentelemetry/instrumentation-winston");

const client = new ApplicationInsightsClient({})
// Get global tracerProvider registered during ApplicationInsightsClient initialization
const tracerProvider = trace.getTracerProvider().getDelegate(); 
registerInstrumentations({
    instrumentations: new WinstonInstrumentation(),
    tracerProvider: tracerProvider
});
```

**Comment by mderriey** [comment]
Thanks @hectorhdzg, apologies for replying a bit late, I've moved on to another project.

Your solution worked nicely, thanks 🎉
It's nice to see App Insights / Azure Monitor SDKs come closer to vanilla OTel.

In case it helps others, when using TypeScript, I had to slap a couple type casts to get the tracer provider, i.e.:

```ts
const tracerProvider = (trace.getTracerProvider() as ProxyTracerProvider).getDelegate() as NodeTracerProvider
```

That was also useful to add span processors, as `addSpanProcessor` is not available on the base `TracerProvider` interface.
